docs/data-management.rst

```diff
@@ -280,38 +426,78 @@ Step-by-Step
+   Key-Value pairs and FileAnnotations, but Comments and Ratings
+   will be duplicated. This can be useful to ensure that a single
+   Comment is not attached to multiple objects which might cause confusion
+   when the comment is edited. The duplicated Annotations
+   and the two Projects with the linked Datasets and Images will belong
+   to the current user (`user-1`), even if all the original Annotations
+   may belong to other users in the read-annotate group. Run::
 
    $ omero logout
    $ omero login -u user-1 -g read-annotate-group
    $ omero duplicate Project:$ID1,$ID2 --reference Annotation --duplicate CommentAnnotation,LongAnnotation --report
 
-#. Duplicate two Projects of another user in read-only group type. The group name in our example below is `read-only-group`. If the duplicator is not an administrator, administrator with restricted privileges or group owner, they cannot link the annotations of another user to their duplicate in a read-only group. They might duplicate all the annotations or exclude the duplication of all the annotations by excluding the Link duplication to the relevant objects as shown below. Run::
+#. Duplicate two Projects of another user in read-only group type.
+   The group name in our example below is `read-only-group`. 
+   If the duplicator is not an administrator,
+   administrator with restricted privileges or group owner,
+   they cannot link the annotations of another user to their duplicate
+   in a read-only group. They might duplicate 
+   all the annotations (this is the default behaviour)
+   or exclude the duplication of all the annotations by excluding
+   the Link duplication to the relevant objects as shown below. Run::
 
    $ omero logout
    $ omero login -u user-1 -g read-only-group
    $ omero duplicate Project:$ID1,$ID2 --ignore IAnnotationLink,Roi --report
 
 .. note::
-    You must log in to the group where the data are, either by virtue of this group being your default group or by using the `-g` flag as shown in the examples above, otherwise the `omero-cli-duplicate` plugin will not find the data. This is a current limitation.
-
-    If you intend to move the duplicate into a different group, it is recommended that you duplicate the annotations as well. If you link the annotations from other objects to your duplicates, the link might be cut during the subsequent move of that duplicate to another group.
+    You must log in to the group where the data are,
+    either by virtue of this group being your default group or
+    by using the `-g` flag as shown in the examples above,
+    otherwise the `omero-cli-duplicate` plugin will not find the data.
+    This is a current limitation.
+
+    If you intend to move the duplicate into a different group,
+    it is recommended that you duplicate the annotations as well.
+    If you link the annotations from other objects to your
+    duplicates, the link might be cut during the subsequent
```

Comment:
We've been deleting, cutting, severing links, maybe more. Interesting to try to balance pleasantly readable prose against the technical question of if these acts are identical.

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/17/commits/f55e77faa6c5efbb852ab47e39d5190b52340ad8